tools/licenses/lib/licenses.dart

```diff
@@ -752,6 +752,13 @@ Iterable<_LicenseMatch> _tryReferenceByFilename(String body, LicenseFileReferenc
       if (match.getCopyrights() == '') {
         yield new _LicenseMatch(template, match.start, match.end, debug: '_tryReferenceByFilename (with failed copyright search) looking for $filename');
       } else {
+        License notice = new License.unique(body.substring(match.start, match.end), LicenseType.unknown, origin: origin);
+        yield new _LicenseMatch(
+            new License.unique(body.substring(match.start, match.end), LicenseType.unknown, origin: origin),
+            match.start,
+            match.end,
+            debug: '_tryReferenceByFilename (with successful copyright search) looking for $filename',
+            isDuplicate: true);
         yield* _expand(template, match.getCopyrights(), match.start, match.end, debug: '_tryReferenceByFilename (with successful copyright search) looking for $filename', origin: origin);
```

Comment:
I would have expected this line to throw in the License.expand logic. The OpenSSL license should be a type that doesn't like being expanded.

This should just be the same as the no-copyright yield (in fact I would just always do that yield).